src/components/modal/iframe/modal.jsx

```diff
@@ -0,0 +1,47 @@
+var classNames = require('classnames');
+var React = require('react');
+
+var Modal = require('../base/modal.jsx');
+
+require('./modal.scss');
+
+Modal.setAppElement(document.getElementById('view'));
+
+var IframeModal = React.createClass({
+    propTypes: {
+        componentKey: React.PropTypes.string,
+        isOpen: React.PropTypes.bool,
+        onRequestClose: React.PropTypes.func,
+        className: React.PropTypes.string,
+        componentRef: React.PropTypes.string,
+        src: React.PropTypes.string
+    },
+    getDefaultProps: function () {
+        return {
+            className: '',
+            iframeClassName: ''
+        };
+    },
+    render: function () {
+        var iframeClasses = classNames(
+            'modal-content-iframe',
+            this.props.className
+        );
+        return (
+            <Modal
+                key={this.props.componentKey}
```

Comment:
I think you're right – I think I was getting caught up in the [`ref` issue](https://github.com/LLK/scratch-www/blob/develop/src/views/splash/splash.jsx#L84) and was doing the same for key, so I'll take that out.